Content.Server/Explosion/EntitySystems/TriggerSystem.cs

```diff
@@ -118,8 +122,30 @@ public void HandleTimerTrigger(EntityUid uid, EntityUid? user, float delay , flo
 
             if (user != null)
             {
-                _adminLogger.Add(LogType.Trigger,
-                    $"{ToPrettyString(user.Value):user} started a {delay} second timer trigger on entity {ToPrettyString(uid):timer}");
+                // There's gotta be a better way to do this, very sussy
+                if (TryComp(uid, out ContainerManagerComponent? grenade) && grenade.TryGetContainer("payload", out IContainer? container))
+                {
```

Comment:
Use container system, also check for containerslot probably if it is one.

Reply:
Please take a look again whenever you're willing, I think this is what you meant